index.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <title>Eternalgratis</title>
+</head>
+<body>
+    <header>
+       <div class="header">
+           <a href="index.html">Welcome</a>
+           <img src="images/menu.png" alt="">
+       </div> 
+    </header>
+    <section class="landing-page">
+        <div class="lp-container">
+            <h1>Hey there. I'm Titilope. <br>
+              <span class="tint">I'm a software developer.</span>
+            </h1>  
+            <p class="lp-text text"> I can help you build a product, feature or website Look through 
+                some of my work and experience! If you like what you see and have a project you need coded, 
+                don't hestiate to contact me.
+            </p>
+            <div class="media-links">
+                <ul class="social">
+                    <li><a href=""><img src="images/twitter.svg" alt="my twitter profil"></a></li>
+                    <li><a href=""><img src="images/linked.svg" alt="connect on linkedin"></a></li>
+                    <li><a href=""><img src="images/medium.svg" alt="check medium"></a></li>
+                    <li><a href=""><img src="images/github.svg" alt="github profile"></a></li>
+                    <li><a href=""><img src="images/angel.svg" alt="angel list"></a></li>
+                </ul>
+                </div>
+            </div>
+    </section>
+
+    <section id="works-section">
+        <div class="header2">
+            <a class="arrow" href=""><img src="./images/dropdown.png" alt="dropdown icon"></a>
+            <h3 class="header-text">My Recent Works</h3>
+               <div class="separator"></div>
+        </div>
+    
+        <div class="Projects">
+                <div class="Proj-text">
+                    <h3>Multi-Post Stories</h3>
+                    <h3>Gain+Glory</h3>
+                    <ul class="pro-tech">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+                    <br>
+                    <a class="pro-btn" href="#works-section">See Project</a>
+                </div>
+            </div>
```

Comment:
- [ ] [Required] Please make sure to add a flexbox on the card view of the individual project element.
![Card_Screenshot from 2022-05-29 03-24-00](https://user-images.githubusercontent.com/24191094/170846939-84639c4b-6533-411a-b1e9-683b4a407fb9.png)

Reply:
I just send my corrected project. Kindly review it for me. Thank you